test/PR_test/unit_test/op/numpyop/univariate/test_color.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2021 The FastEstimator Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import unittest
+
+import numpy as np
+
+from fastestimator.op.numpyop.univariate import Color
+
+
+class TestColor(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.single_input = [255 * np.random.rand(28, 28, 3).astype(np.uint8)]
+        cls.single_output_shape = (28, 28, 3)
+        cls.multi_input = [
+            255 * np.random.rand(28, 28, 3).astype(np.uint8), 255 * np.random.rand(28, 28, 3).astype(np.uint8)
+        ]
+        cls.multi_output_shape = (28, 28, 3)
+
+    def test_single_input(self):
+        color = Color(inputs='x', outputs='x')
+        output = color.forward(data=self.single_input, state={})
+        with self.subTest('Check output type'):
+            self.assertEqual(type(output), list)
+        with self.subTest('Check output image shape'):
+            self.assertEqual(output[0].shape, self.single_output_shape)
```

Comment:
these tests aren't really validating much of anything...

Reply:
True. I'll write some tests specific for these ops. 
However on another note, this is the trend for almost all the NumpyOps as well.